src/templates/playgroundTemplate.ts

```diff
@@ -1,27 +1,15 @@
 const template: string = `// Select the database to use.
-use("test");
-
-// Insert a few documents.
-db.sales.insertMany([
-  { "_id" : 1, "item" : "abc", "price" : 10, "quantity" : 2, "date" : new Date("2014-03-01T08:00:00Z") },
-  { "_id" : 2, "item" : "jkl", "price" : 20, "quantity" : 1, "date" : new Date("2014-03-01T09:00:00Z") },
-  { "_id" : 3, "item" : "xyz", "price" : 5, "quantity" : 10, "date" : new Date("2014-03-15T09:00:00Z") },
-  { "_id" : 4, "item" : "xyz", "price" : 5, "quantity" :  20, "date" : new Date("2014-04-04T11:21:39.736Z") },
-  { "_id" : 5, "item" : "abc", "price" : 10, "quantity" : 10, "date" : new Date("2014-04-04T21:23:13.331Z") },
-  { "_id" : 6, "item" : "def", "price" : 7.5, "quantity": 5, "date" : new Date("2015-06-04T05:08:13Z") },
-  { "_id" : 7, "item" : "def", "price" : 7.5, "quantity": 10, "date" : new Date("2015-09-10T08:43:00Z") },
-  { "_id" : 8, "item" : "abc", "price" : 10, "quantity" : 5, "date" : new Date("2016-02-06T20:20:13Z") },
-]);
+use('test');
 
 // Run a find command.
-db.sales.find({ date: { $gte: new Date("2014-04-04"), $lt: new Date("2014-04-05") } });
+db.myCollection.find({ foo: 'bar' });
 
 // Run an aggregation.
-const aggregation = [
-  { $match: { date: { $gte: new Date("2014-01-01"), $lt: new Date("2015-01-01") } } },
-  { $group: { _id : "$item", totalSaleAmount: { $sum: { $multiply: [ "$price", "$quantity" ] } } } }
+const agg = [
```

Comment:
Maybe aggPipeline? agg is good too.

Reply:
This is just a revert of a commit that should not have been in this PR